From 4ba76ed7c8d2889191eaadc338b7b23e0e1284ad Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 14 May 2018 11:17:25 -0700
Subject: [PATCH] TRITON-382 binder update in MANTA-3676 broken binder usage in
 Triton: "config.staticLabels.instance (string) is required" Reviewed by: Alex
 Wilson <alex.wilson@joyent.com> Reviewed by: Robert Bogart
 <Robert.bogart@joyent.com> Approved by: Alex Wilson <alex.wilson@joyent.com>

---
 sapi_manifests/binder/template | 10 +++++++---
 1 file changed, 7 insertions(+), 3 deletions(-)

diff --git a/sapi_manifests/binder/template b/sapi_manifests/binder/template
index beb7a06..9b0efbc 100644
--- a/sapi_manifests/binder/template
+++ b/sapi_manifests/binder/template
@@ -1,4 +1,6 @@
 {
+    {{! "dns_domain" means this is Triton (vs Manta).
+        Only render recursion block in Triton. }}
     {{#dns_domain}}
     "dnsDomain": "{{{dns_domain}}}",
     "datacenterName": "{{{datacenter_name}}}",
@@ -22,13 +24,15 @@
             "clientTimeout": 120000,
             "connectTimeout": 3000
         }
-    }
+    },
     {{/dns_domain}}
     {{^dns_domain}}
     "dnsDomain": "{{{DNS_DOMAIN}}}",
     "datacenterName": "{{{DATACENTER}}}",
+    {{/dns_domain}}
+
+    {{! Metrics labels values. }}
     "instance_uuid": "{{auto.ZONENAME}}",
     "server_uuid": "{{auto.SERVER_UUID}}",
     "service_name": "{{SERVICE_NAME}}"
-    {{/dns_domain}}
-}
+}
\ No newline at end of file
-- 
2.21.0

